osidb/models.py

```diff
@@ -1291,6 +1319,11 @@ def fix_state(self):
                 return Affect.AffectFix.DEFER
         return Affect.AffectFix.AFFECTED
 
+    @property
+    def is_closed(self):
+        if self.status:
+            return self.status.upper() == "CLOSED"
```

Comment:
`status` values are set to `null=False` and `blank=False` so if condition seems kinda redundant. Also in cases where `status` would allow blank fields this would ultimately result in returning `None` which doesn't seem right as property with naming convention `is_xxx` denotes the `bool` expectations.

Reply:
Also, what is the purpose of this property? I don't see it used anywhere.

Reply:
Right, given that, the `exclude`s in the validations should do an ILIKE instead of an exact match

Reply:
I have modified the filters as suggested, created new validations in trackers additional to the ones in affects and changed its behaviors for blocking validations (ValidationError) as discussed in GChat.

Reply:
This is still constructed that it may return `None` in some cases which is undesirable.

Reply:
Makes sense, removed condition.